libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/OnOfflineRouteFoundCallback.kt

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.navigation.route.onboard
+
+import com.mapbox.navigation.base.route.model.Route
+import com.mapbox.navigation.route.onboard.model.OfflineError
+
+/**
+ * Callback used for finding offline routes.
+ */
+interface OnOfflineRouteFoundCallback {
```

Comment:
This is only used internally from `MapboxOnboardRouter#getRoute`. Should we add the `internal` modifier?

Reply:
yep, done